test/journeys/specs/widget-message-meet/messaging.js

```diff
@@ -0,0 +1,92 @@
+/* eslint-disable max-nested-callbacks */
+
+import {assert} from 'chai';
+
+import testUsers from '@ciscospark/test-helper-test-users';
+import '@ciscospark/plugin-conversation';
+import CiscoSpark from '@ciscospark/spark-core';
+import waitForMercuryEvent from '../../lib/wait-for-mercury-event';
+import waitForPromise from '../../lib/wait-for-promise';
+
+describe(`Widget Message Meet`, function() {
+  if (process.env.DEBUG_JOURNEYS) {
+    console.warn(`Running with DEBUG_JOURNEYS may require you to manually kill wdio`);
+    const timeout = 10 * 60 * 1000;
+    // eslint-disable-next-line no-invalid-this
+    this.timeout(timeout);
+    // Leaves the browser open for further testing and inspection
+    after(() => {
+      browser.timeouts(`implicit`, 1200000);
+      return browser.waitUntil(() => false, 1200000, `done waiting: bye`, 10000);
+    });
+  }
+
+  let mccoy, spock;
+  process.env.CISCOSPARK_SCOPE = [
+    `webexsquare:get_conversation`,
```

Comment:
could've sworn i replied to this...this scope is the old internal scope. I'm not sure what checks it at this point, but it's what we've got in all the other tests; i don't want to get rid of it until we're absolutely certain we can.